tool/bots.sh

```diff
@@ -73,7 +73,7 @@ export PATH="$PATH":"$DEVTOOLS_DIR/tool/bin"
 devtools_tool pub-get --only-main
 
 # Generate code.
-bash tool/generate_code.sh
+bash tool/bin/devtools_tool generate-code
```

Comment:
this can just be `devtools_tool generate-code`
see the above command

Reply:
Oops, should've noticed that. Fixed, thanks!